src/base/net/downloadmanager.cpp

```diff
@@ -498,7 +491,7 @@ namespace
         }
 
         // Resolve relative urls
-        const QUrl resolvedUrl = (newUrl.isRelative()) ? m_reply->url().resolved(newUrl) : newUrl;
+        const QUrl resolvedUrl = (newUrl.isRelative() ? m_reply->url().resolved(newUrl) : newUrl);
```

Comment:
I think we can omit the outer parentheses.

Reply:
You usually insist on the opposite...

Reply:
Really I don't like it here so I'll remove it.